index.html

```diff
@@ -30,7 +30,7 @@ <h2>Packages</h2>
         <div class="col-md-4">
             <h2>Admin</h2>
             <p>
-                <a href="admin.html" class="btn btn-primary">
+                <a href="./admin" class="btn btn-primary">
```

Comment:
`./` can be dropped

Reply:
... or is it needed for base-url resolution?

Reply:
I wanted the href attributes to resemble one another `./`, `../`, `./admin` make it obvious that we are dealing with relative path that are resolved against something.

Reply:
Ok, thanks.